edi/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,26 @@
+This module is intended to be inherited by other modules in order to define the proper
```

Comment:
yes and no... the aim is also to allow to not have to code anything and just work w/ configuration